### ISSUE TITLE
Typo in sunfish_nnue.py lines 217 and  261

### ISSUE THREAD

**Issue by foo123** [thomasahle/sunfish#120]
Line 217 is:

`old_ep, old_kp, old_wc, old_bc = self.ep, self.kp, self.ec, self.bc`

instead of `self.wc` (Position field) it has  `self.ec` (no such field)

similar for line 261

**Comment by thomasahle** [comment]
Good point. The nnue version just isn't working well yet. But this would be good to fix.

**Comment by foo123** [comment]
I have made a javascript port of sunfish: [sunfish.js](https://github.com/foo123/sunfish.js) and have ported both `sunfish.py` and `sunfish_nnue.py` versions. 

I have also added `maxdepth`  (eg `go depth x`) and `stop` uci options since they are needed in js. Other than that (and possibly some typos) I left the code as  similar to `py` code as possible.

**Comment by professor314** [comment]
Hey, I noticed this has been open a while. I went ahead and submitted a fix in PR #133. It's a two character change on lines 217 and 261. Let me know if there's anything I missed.

**Comment by thomasahle** [comment]
Thanks @foo123 for the careful report and @professor314 for PR #133 — and apologies that events overtook the PR: the NNUE engine has moved off master to the [nnue branch](https://github.com/thomasahle/sunfish/tree/nnue-mutable-board) until it's fast enough to be worth using. The mutable-board rewrite on that branch replaced both affected lines, so `self.ec` no longer exists there (verified by grep). Closing as fixed on the branch.

**Comment by foo123** [comment]
@thomasahle maybe using [self-play algorithm](https://en.wikipedia.org/wiki/Self-play), appropriately adjusted, can help in training the NNUE better
